prometheus-exporters/vrops-exporter/templates/collector-configmap.yaml

```diff
@@ -31,6 +31,8 @@ data:
         - label: "cpu_demand_percentage"
           statkey: "cpu|demandPct"
         - label: "cpu_usage_average_percentage"
+        - label: "cpu_demand_megahertz"
+          statkey: "cpu|demandmhz"
           statkey: "cpu|usage_average"
         - label: "memory_host_usage_kilobytes"
```

Comment:
There is a mistake. You added your metric between a `label` and `statkey` entry.

Reply:
Thanks Richard, I just corrected it. Please review.